LICENSE-3rdparty.csv

```diff
@@ -1,177 +1,951 @@
-core,github.com/patrickmn/go-cache,MIT
-core,github.com/pborman/uuid,BSD-3-Clause
-core,github.com/pelletier/go-toml,MIT
-core,github.com/pierrec/lz4,BSD-3-Clause
-core,github.com/pkg/errors,BSD-2-Clause
-core,github.com/pmezard/go-difflib,BSD-3-Clause
-core,github.com/prometheus/client_golang,Apache-2.0
-core,github.com/prometheus/client_model,Apache-2.0
-core,github.com/prometheus/common,Apache-2.0
-core,github.com/prometheus/procfs,Apache-2.0
-core,github.com/samuel/go-zookeeper,BSD-3-Clause
-core,github.com/shirou/gopsutil,BSD-3-Clause
-core,github.com/shirou/w32,BSD-3-Clause
-core,github.com/sirupsen/logrus,MIT
-core,github.com/spf13/afero,Apache-2.0
-core,github.com/spf13/cast,MIT
-core,github.com/spf13/cobra,Apache-2.0
-core,github.com/spf13/jwalterweatherman,MIT
-core,github.com/spf13/pflag,BSD-3-Clause
-core,github.com/stretchr/objx,MIT
-core,github.com/stretchr/testify,MIT
-core,github.com/syndtr/gocapability,BSD-2-Clause
-core,github.com/ugorji/go,MIT
-core,github.com/ulikunitz/xz,BSD-3-Clause
-core,github.com/urfave/negroni,MIT
-core,golang.org/x/crypto,BSD-3-Clause
-core,golang.org/x/mobile,BSD-3-Clause
-core,golang.org/x/net,BSD-3-Clause
-core,golang.org/x/sys,BSD-3-Clause
-core,golang.org/x/sync,BSD-3-Clause
-core,golang.org/x/text,BSD-3-Clause
-core,golang.org/x/time,BSD-3-Clause
-core,google.golang.org/genproto,Apache-2.0
-core,google.golang.org/grpc,Apache-2.0
-core,gopkg.in/Knetic/govaluate.v3,MIT
-core,gopkg.in/inf.v0,BSD-3-Clause
-core,gopkg.in/natefinch/lumberjack.v2,MIT
-core,gopkg.in/yaml.v2,Apache-2.0
-core,gopkg.in/zorkian/go-datadog-api.v2,BSD-3-Clause
-core,k8s.io/api,Apache-2.0
-core,k8s.io/apimachinery,Apache-2.0
-core,k8s.io/apiserver,Apache-2.0
-core,k8s.io/client-go,Apache-2.0
-core,k8s.io/kube-openapi,Apache-2.0
-core,k8s.io/metrics,Apache-2.0
-core,k8s.io/utils,Apache-2.0
-core,k8s.io/cri-api,Apache-2.0
-core,github.com/opencontainers/go-digest,Apache-2.0
-core,github.com/philhofer/fwd,MIT
-core,github.com/DataDog/datadog-go,MIT
-core,github.com/tinylib/msgp,MIT
-core,sigs.k8s.io/structured-merge-diff,Apache-2.0
-core,sigs.k8s.io/controller-runtime,Apache-2.0
-core,github.com/grpc-ecosystem/go-grpc-prometheus,Apache-2.0
-core,github.com/vito/go-sse,Apache-2.0
-core,code.cloudfoundry.org/cfhttp,Apache-2.0
-core,github.com/bhmj/jsonslice,MIT
-core,code.cloudfoundry.org/lager,Apache-2.0
-core,code.cloudfoundry.org/bbs,Apache-2.0
-core,github.com/tedsuo/rata,MIT
-core,github.com/bmizerany/pat,MIT
-core,code.cloudfoundry.org/tlsconfig,Apache-2.0
-process,github.com/DataDog/gopsutil,BSD-3-Clause
-network,github.com/iovisor/gobpf,Apache-2.0
-network,github.com/florianl/go-conntrack,MIT
-network,github.com/mdlayher/netlink,MIT
-network,github.com/google/gopacket,BSD-3-Clause
-network,github.com/DataDog/mmh3,MIT
-network,github.com/vishvananda/netns,Apache-2.0
-network,github.com/miekg/dns,BSD-3-Clause
-network,golang.org/x/tools,BSD-3-Clause
+core,"code.cloudfoundry.org/bbs",Apache-2.0
```

Comment:
Note: because we are per-pkg instead of per-project here now, we'll have way more often too update the license file.